includes/class-dispatcher.php

```diff
@@ -395,6 +395,13 @@ protected static function should_send_to_followers( $activity, $actor, $outbox_i
 			in_array( $actor->get_followers(), $audience, true )
 		);
 
+		if ( $send ) {
+			$followers = Followers::get_inboxes_for_activity( $activity->to_json(), $actor->get__id() );
```

Comment:
what about using `Followers::count_followers` instead?

Reply:
That doesn't account for potentially adding blog user followers, that get_inboxes_for_activity() does.

FWIW, both follower lists should be cached on sites with object cache

Reply:
makes sense...